specification/dartLangSpec.tex

```diff
@@ -3282,10 +3285,11 @@ \subsubsection{The Operator `=='}
 \end{itemize}
```

Comment:
If I'm going to nit, `bool` instances likely override `hashCode`.

"evaluation of a list literal", should it be "of a constant list literal, ... a constant map literal ..., or a constant set literal"?

Reply:
Adjusted the text to specify directly that `bool`, const-able function closurizations, and collection literals have primitive equality. Then we don't need to worry about whether or not `bool` overrides `hashCode`.